jazz_environments/index.js

```diff
@@ -27,189 +27,204 @@ const logger = require("./components/logger.js"); //Import the logging module.
 const utils = require("./components/utils.js")(); //Import the utils module.
 const crud = require("./components/crud")(); //Import the utils module.
 const request = require("request");
-const async = require("async");
 const _ = require("lodash");
 const validateUtils = require("./components/validation")();
+var  errorHandler = errorHandlerModule();
 
 var handler = (event, context, cb) => {
     //Initializations
-    var errorHandler = errorHandlerModule();
+    // var errorHandler = errorHandlerModule();
     var config = configObj(event);
     logger.init(event, context);
     logger.info("event:" + JSON.stringify(event));
     global.config = config;
+    global.userId = event.principalId;
+    global.authorization = event.headers.Authorization;
+    var env_tableName = global.config.services_environment_table;
+    var indexName = global.config.services_environment_index;
 
     try {
+        
+
+        genericInputValidation(event)
+        .then(function(result){
+            logger.info("Valid event params");
```

Comment:
this is at best "debug" or just remove it.

Reply:
Using debug